src/components/TreeView.js

```diff
@@ -56,6 +97,14 @@ function RowValue(props) {
     separator = ': ';
   }
 
+  if (position === 'TransactionEnvelope.signatures') {
+    return formatSignatureCheckState(node, separator)
+  }
+
+  if (position.match(/(\.signature)+$/)) {
```

Comment:
We really need a full [regexp](https://regex101.com/r/m4xxKw/1) here. No idea how `TransactionEnvelope` will looks like in a future and it will break if there's another `signature` field.